.eslintignore

```diff
@@ -7,6 +7,7 @@ external/webL10n/
 external/cmapscompress/
 external/builder/fixtures/
 external/builder/fixtures_esprima/
+src/scripting_api/quickjs-eval.js
```

Comment:
Unless we can somehow add an external dependency, rather than simply dumping the code into PDF.js repository, anything *specific* to quickjs should probably go into a `external/quickjs` (or similar) folder instead.